wifi/wpa_supplicant_8_lib/Android.mk

```diff
@@ -0,0 +1,61 @@
+#
```

Comment:
Instead of having it under wifi we can directly have it under wpa_supplicant_8_lib